<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinesStartWidget</class>
 <widget class="QWidget" name="MinesStartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="MinesSpinBox" name="widget_horizontal" native="true"/>
   </item>
   <item>
    <widget class="MinesSpinBox" name="widget_vertical" native="true"/>
   </item>
   <item>
    <widget class="MinesSpinBox" name="widget_mines" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_playGame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#ffffff</string>
     </property>
     <property name="text">
      <string>&amp;Play Game</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MinesSpinBox</class>
   <extends>QWidget</extends>
   <header>minesspinbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
